### ISSUE TITLE
Add ability to add SVG based shapes

### ISSUE THREAD

**Issue by atapas** [TryShape/tryshape#113]
### Discussed in https://github.com/TryShape/tryshape/discussions/75

<div type='discussions-op-text'>

<sup>Originally posted by **atapas** July  3, 2021</sup>
So far, we can create polygonal, circular, and elliptical shapes. We need to add the abilities to create curvy corner shapes like a Heart shape.</div>

**Comment by manastelavane** [comment]
So, we need to add new heart shape, right?

**Comment by atapas** [comment]
> So, we need to add new heart shape, right?

Not really, right now the shapes are clip-path based, We want to add additional functionality for users to add SVG based circular, elliptical shapes

**Comment by Waishnav** [comment]
Hey, could you please provide any resources to understand the SVG format better way? 
So that I can understand the issues and features that we want to implement.

**Comment by akshayw1** [comment]
Is this issue still open, so that I can work on this.
